myapp/src/day2/Question10/UI.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { MyContext } from './LanguageSwitcher'
+import { translations } from './data'
+
+const UI = () => {
+  const { language } = useContext(MyContext)
+  const [content, setContent] = useState(null)
+
+  useEffect(() => {
+    const filterTranslation = translations.filter(
+      (translation) => translation.language === language,
+    )
+    setContent(filterTranslation[0])
+  }, [language])
+
+  return (
+    <div className="content">
```

Comment:
language switcher not working as expected

Reply:
![Screenshot from 2024-04-09 10-53-35](https://github.com/sumitnegikv369/REACT-Training/assets/161808037/e10976d3-6ff9-49e1-8160-5b1cb873bbd0)
in my system i am able to switch in between of different language,  Could you please provide more details on the specific aspect of this functionality that is not functioning as expected?

Reply:
updated code: [ onClick event to onChange ]